src/inverseRobot.js

```diff
@@ -24,7 +24,34 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  if (Object.keys(robot).length === 0) {
+    return {};
+  }
```

Comment:
it's actually much simpler - just declare a blank object variable, say `newRobot = {}`
and you return it from the function - so if no changes will happen it remains a blank object.

And you can fill that object with a `for ( ... in ... )` loop - check if your newRobot hasOwnProperty of an original robot:
- `robot[key]` - should become a key,
- and a `key` - should become a value